staging/src/k8s.io/apiextensions-apiserver/pkg/controller/openapi/builder.go

```diff
@@ -96,33 +96,39 @@ func BuildSwagger(crd *apiextensions.CustomResourceDefinition, version string) (
 
 	routes := make([]*restful.RouteBuilder, 0)
 	root := fmt.Sprintf("/apis/%s/%s/%s", b.group, b.version, b.plural)
+
+	var params []*restful.Parameter
+
 	if b.namespaced {
-		routes = append(routes, b.buildRoute(root, "", "GET", "list", sampleList).
-			Operation("list"+b.kind+"ForAllNamespaces"))
+		namespaceParam := b.ws.PathParameter("namespace", "object name and auth scope, such as for teams and projects").DataType("string")
+		routes = append(routes, b.buildRoute(root, "", "GET", "list", "list", sampleList, params).Operation("list"+b.kind+"ForAllNamespaces"))
+		params = append(params, namespaceParam)
 		root = fmt.Sprintf("/apis/%s/%s/namespaces/{namespace}/%s", b.group, b.version, b.plural)
 	}
-	routes = append(routes, b.buildRoute(root, "", "GET", "list", sampleList))
-	routes = append(routes, b.buildRoute(root, "", "POST", "create", sample).Reads(sample))
-	routes = append(routes, b.buildRoute(root, "", "DELETE", "deletecollection", status))
+	routes = append(routes, b.buildRoute(root, "", "GET", "list", "list", sampleList, params))
+	routes = append(routes, b.buildRoute(root, "", "POST", "create", "create", sample, params).Reads(sample))
```

Comment:
`actionVerb` cannot be "create". See https://github.com/kubernetes/kubernetes/pull/80074#discussion_r311811636

Reply:
oh good catch